tests/unit_tests/steps/test_root_inc_files.py

```diff
@@ -0,0 +1,47 @@
+from pathlib import Path
+from unittest import mock
+
+import pytest
+
+from fab.constants import BUILD_OUTPUT
+from fab.steps.root_inc_files import RootIncFiles
+
+
+@pytest.fixture
+def common_data():
+    source_root = Path('/foo/source')
+    build_output = source_root.parent / BUILD_OUTPUT
+    step = RootIncFiles(source_root=source_root)
+    return source_root, build_output, step
+
+
+class TestRootIncFiles(object):
+
+    def test_vanilla(self, common_data):
+        # ensure it copies the inc file
+        source_root, build_output, step = common_data
+        inc_files = [Path('/foo/source/bar.inc')]
+
+        with mock.patch('fab.steps.root_inc_files.shutil') as mock_shutil:
+            step.run(artefacts={'all_source': inc_files}, config=None)
+
+        mock_shutil.copy.assert_called_once_with(inc_files[0], source_root.parent / BUILD_OUTPUT)
```

Comment:
Is there more which could be tested here? Does `step` have any getters which could be tested? This applies to the other tests too.

Reply:
Not at this time, if that's ok. We're aiming for little and often.